### PR TITLE
Add Keywords and Categories

### PR DESCRIPTION
**Pull request by jcape**
- WIP
- Sort package section, add [keywords](https://crates.io/keywords) and [categories](https://crates.io/categories)

### CODE REVIEW

File: tstdc/sys/Cargo.toml [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 name = "mc-sgx-tstdc-sys"
 version = "0.1.0"
+authors = ["MobileCoin"]
+categories = ["external-ffi-bindings", "hardware-support", "no-std"]
+description = "FFI bindings for `sgx_tstdc`."
 edition = "2021"
-rust-version = "1.62.1"
+keywords = ["ffi", "sgx", "no-std"]
 license = "Apache-2.0"
+links = "sgx_tstd"
```
Comment (nick-mobilecoin) [inline tstdc/sys/Cargo.toml:10]:
```suggestion
links = "sgx_tstdc"
```

File: tstdc/sys/types/Cargo.toml [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 name = "mc-sgx-tstdc-sys-types"
 version = "0.1.0"
+authors = ["MobileCoin"]
+categories = ["external-ffi-bindings", "hardware-support", "no-std"]
+description = "FFI types for `sgx_tservice`."
 edition = "2021"
+keywords = ["ffi", "sgx", "no-std"]
 license = "Apache-2.0"
-rust-version = "1.62.1"
+links = "sgx_tstdc"
```
Comment (nick-mobilecoin) [inline tstdc/sys/types/Cargo.toml:10]: Don't think the types should link to the library
```suggestion
```

File: tstdc/sys/types/Cargo.toml [hunk 3]
```diff
@@ -1,17 +1,17 @@
 [package]
 name = "mc-sgx-tstdc-sys-types"
 version = "0.1.0"
+authors = ["MobileCoin"]
+categories = ["external-ffi-bindings", "hardware-support", "no-std"]
+description = "FFI types for `sgx_tservice`."
```
Comment (nick-mobilecoin) [inline tstdc/sys/types/Cargo.toml:6]:
```suggestion
description = "FFI types for `sgx_tstdc`."
```

File: urts/sys/Cargo.toml [hunk 4]
```diff
@@ -1,21 +1,28 @@
 [package]
 name = "mc-sgx-urts-sys"
 version = "0.1.0"
+authors = ["MobileCoin"]
+categories = ["external-ffi-bindings", "hardware-support"]
+description = "FFI bindings for `sgx_urts`."
 edition = "2021"
+keywords = ["ffi", "sgx"]
+license = "Apache-2.0"
```
Comment (nick-mobilecoin) [inline urts/sys/Cargo.toml:9]: Do we want links for the `so`s?
The docs, https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key, just say "native library", which can be either.
```suggestion
license = "Apache-2.0"
links = "sgx_urts"
```

File: Cargo.toml [hunk 5]
```diff
@@ -5,8 +5,8 @@ members = [
     "capable/sys/types",
     "core/sys/types",
     "core/build",
-    "tcrypto/sys/types",
     "tcrypto/sys",
+    "tcrypto/sys/types",
     "panic/abort",
     "quote_verify",
```
Comment (nick-mobilecoin) [inline Cargo.toml:11]: nit perhaps fully alphebetize
```suggestion
    "panic/abort",
    "quote_verify",
    "tcrypto/sys",
    "tcrypto/sys/types",
```

**Review by nick-mobilecoin — APPROVED** [review]
LGTM, sans a few tweaks

**Review by awygle — APPROVED** [review]
:+1:

**Pull request merged**